### PR TITLE
LinwoodCloud.Butterfly version 1.0.0-rc.3

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34656)

### CODE REVIEW

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 1.0.0-rc.2
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0
```

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 1.0.0-rc.2
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.installer.yaml:5]:
```suggestion
PackageVersion: 1.0
```

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 1.0.0-rc.2
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.yaml:5]:
```suggestion
PackageVersion: 1.0
```

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 1.0.0-rc.2
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://github.com/LinwoodCloud/butterfly/releases/download/stable/Butterfly-Setup.exe
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.installer.yaml:9]:
```suggestion
  InstallerUrl: https://github.com/LinwoodCloud/butterfly/releases/download/v1.0.0-rc.2/Butterfly-Setup.exe
```

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.installer.yaml:5]: `PackageVersion` must match Control Panel.

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.locale.en-US.yaml:5]: `PackageVersion` must match Control Panel.

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 1.0.0-rc.2
+PackageLocale: en-US
+Publisher: LinwoodCloud
+PackageName: Butterfly
+License: GPL-3.0
+ShortDescription: Butterfly Setup
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Butterfly is a note app where your ideas come first. You can paint, add texts and export them easily on every device.
```

File: manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/l/LinwoodCloud/Butterfly/1.0.0-rc.2/LinwoodCloud.Butterfly.yaml:5]: `PackageVersion` must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You also have to rename the version folder by pressing `.` (dot) on your keyboard on this PR which will automatically bring up Visual Studio Code.

Please change the version folder from `1.0.0-rc.2` to `1.0` so that the validation will pass.

**Comment by CodeDoctorDE** [comment]
Oh sorry. Can I change the version number to 1.0.0-rc.2 in the project? This would be better that I can publish more release candidates

**Comment by ItzLevvie** [comment]
> Oh sorry. Can I change the version number to 1.0.0-rc.2 in the project? This would be better that I can publish more release candidates

WinGet expects the version to be 1.0, but 1.0.0-rc.2 is higher than 1.0, so you'll end up with upgrade loops.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/34656#issuecomment-968058951

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**